utility/log.py

```diff
@@ -14,14 +14,25 @@
 
     - local file
     - logstash server
+
+Initial Log format will be "datetime - level - message"
+ex: 2022-11-15 11:37:00,346 - DEBUG - cephci.log.py:configure_logger:227 - Completed log configuration
```

Comment:
I think it would be good to display like this `cephci.utility.log.py:<line_num>`.

Current logger displays `cephci.install_prereq.py` where it could be `cephci.tests.misc.install_prereq.py`.
```
2022-11-15 11:37:00,355 - INFO - cephci.install_prereq.py:run:47 - Running test
```

Reply:
Made changes